packages/tx/src/eip1559Transaction.ts

```diff
@@ -174,6 +175,11 @@ export class FeeMarketEIP1559Transaction extends BaseTransaction<FeeMarketEIP155
       maxPriorityFeePerGas: this.maxPriorityFeePerGas,
     })
 
+    const notArray = { ...txData }
+    delete notArray.accessList
+
+    BaseTransaction._validateNotArray(notArray)
```

Comment:
you could just do `BaseTransaction._validateNotArray(txData)` , was typescript giving issues for doing this?